src/languages/sqf.js

```diff
@@ -82,351 +83,394 @@ export default function(hljs) {
-        'triggerAttachObject triggerAttachVehicle triggerDynamicSimulation triggerStatements triggerText ' +
-        'triggerTimeout triggerTimeoutCurrent triggerType turretLocal turretOwner turretUnit tvAdd tvClear ' +
-        'tvCollapse tvCollapseAll tvCount tvCurSel tvData tvDelete tvExpand tvExpandAll tvPicture ' +
-        'tvSetColor tvSetCurSel tvSetData tvSetPicture tvSetPictureColor tvSetPictureColorDisabled ' +
-        'tvSetPictureColorSelected tvSetPictureRight tvSetPictureRightColor tvSetPictureRightColorDisabled ' +
-        'tvSetPictureRightColorSelected tvSetText tvSetTooltip tvSetValue tvSort tvSortByValue tvText ' +
-        'tvTooltip tvValue type typeName typeOf UAVControl uiNamespace uiSleep unassignCurator ' +
-        'unassignItem unassignTeam unassignVehicle underwater uniform uniformContainer uniformItems ' +
-        'uniformMagazines unitAddons unitAimPosition unitAimPositionVisual unitBackpack unitIsUAV unitPos ' +
-        'unitReady unitRecoilCoefficient units unitsBelowHeight unlinkItem unlockAchievement ' +
-        'unregisterTask updateDrawIcon updateMenuItem updateObjectTree useAISteeringComponent ' +
-        'useAudioTimeForMoves userInputDisabled vectorAdd vectorCos vectorCrossProduct vectorDiff ' +
-        'vectorDir vectorDirVisual vectorDistance vectorDistanceSqr vectorDotProduct vectorFromTo ' +
-        'vectorMagnitude vectorMagnitudeSqr vectorModelToWorld vectorModelToWorldVisual vectorMultiply ' +
-        'vectorNormalized vectorUp vectorUpVisual vectorWorldToModel vectorWorldToModelVisual vehicle ' +
-        'vehicleCargoEnabled vehicleChat vehicleRadio vehicleReceiveRemoteTargets vehicleReportOwnPosition ' +
-        'vehicleReportRemoteTargets vehicles vehicleVarName velocity velocityModelSpace verifySignature ' +
-        'vest vestContainer vestItems vestMagazines viewDistance visibleCompass visibleGPS visibleMap ' +
-        'visiblePosition visiblePositionASL visibleScoretable visibleWatch waves waypointAttachedObject ' +
-        'waypointAttachedVehicle waypointAttachObject waypointAttachVehicle waypointBehaviour ' +
-        'waypointCombatMode waypointCompletionRadius waypointDescription waypointForceBehaviour ' +
-        'waypointFormation waypointHousePosition waypointLoiterRadius waypointLoiterType waypointName ' +
-        'waypointPosition waypoints waypointScript waypointsEnabledUAV waypointShow waypointSpeed ' +
-        'waypointStatements waypointTimeout waypointTimeoutCurrent waypointType waypointVisible ' +
-        'weaponAccessories weaponAccessoriesCargo weaponCargo weaponDirection weaponInertia weaponLowered ' +
-        'weapons weaponsItems weaponsItemsCargo weaponState weaponsTurret weightRTD WFSideText wind ',
+        'acctime acos action actionids actionkeys ' +
+        'actionkeysnames actionkeysnamesarray actionname actionparams activateaddons ' +
+        'activatekey add3denconnection add3deneventhandler add3denlayer addaction ' +
```

Comment:
"add3DENEventHandler" vs "add3deneventhandler" I would prefer if you could keep correct casing.
If you need a case-correct export I can give you one.